components/text-utilities/text.css

```diff
@@ -0,0 +1,29 @@
+.text-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    gap: 1rem;
+    background-color: var(--grey);
+    padding:1rem;
+    border-radius: 0.5rem;   
+    margin-top:1rem;
+}
+
+.text{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    gap:1rem;
+    text-align: center;
+}
+
+.grey{
+color:var(--dark-grey);
+}
+
+.text-center{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
```

Comment:
.text and .text-center sound very generic, as you are using flex inside them
maybe write flex-row-center, flex-clmn-center--again these are just examples.